src/main/java/com/semmle/jira/addon/LgtmServlet.java

```diff
@@ -150,6 +152,13 @@ void createIssue(
     issueInputParameters.setIssueTypeId(JiraUtils.getLgtmIssueType().getId());
 
     issueInputParameters.addProperty(Constants.LGTM_PAYLOAD_PROPERTY, rawRequest);
+    CustomFieldManager customFieldManager = ComponentAccessor.getCustomFieldManager();
```

Comment:
I think it would be best to store the ID in the config instead of resolving it dynamically each time it is used.